subscription-service/src/main/kotlin/com/egm/stellio/subscription/service/SubscriptionService.kt

```diff
@@ -176,7 +180,10 @@ class SubscriptionService(
         val subscriptionUpdateInput = parsedInput.first
         val updates = mutableListOf<Mono<Int>>()
 
-        subscriptionUpdateInput.filterKeys { it != "id" && it != "type" && it != "@context" }.forEach {
+        val subscriptionInputWithModifiedAt = parsedInput.first
+            .plus("modifiedAt" to Instant.now().atZone(ZoneOffset.UTC))
+
+        subscriptionInputWithModifiedAt.filterKeys { it != "id" && it != "type" && it != "@context" }.forEach {
```

Comment:
It may worth it to add these 3 "core" properties in JsonLdUtils (on in NgsiLdEntity)

Reply:
I think I did it in another branch, I'll do it, i ll fix the conflicts later